test/install_test.go

```diff
@@ -220,14 +225,21 @@ func TestInject(t *testing.T) {
 
 func TestCheckProxy(t *testing.T) {
 	prefixedNs := TestHelper.GetTestNamespace("smoke-test")
-	out, _, err := TestHelper.LinkerdRun(
-		"check",
-		"--proxy",
-		"--expected-version",
-		TestHelper.GetVersion(),
-		"--namespace",
-		prefixedNs,
-	)
+	var out string
+	err := TestHelper.RetryFor(2*time.Minute, func() error {
```

Comment:
I think we should use a tighter timeout here, since I would consider it a bug if it takes the proxies 2 minutes to reach the ready state. For the other check test, we're using 30 seconds. Mind updating this timeout to match?

Reply:
Per our conversation, these timeouts will be revisited in #1835.